docs_sources/erp_chargeItem-source.adoc

```diff
@@ -105,8 +105,19 @@ Status Codes
 [small]#Innerhalb der vom Server erlaubten Zeitspanne wurde keine gültige Antwort des OCSP-Responders geliefert.#
 |===
 
+== Anwendungsfall Abrechnungsinformation zum Ändern abrufen
+
+.Umsetzungsstand
```

Comment:
In der Überschrift taucht im Namen jetzt noch "Table #" auf. Siehe
![image](https://github.com/gematik/api-erp/assets/10847488/1e2ac76c-0b92-4c6f-9288-2c4932c92943)

Ist das in dem final gerenderten Dokument auch noch so?
https://github.com/gematik/api-erp/blob/mfi/feature/implemented_documentation/docs/erp_chargeItem.adoc

Bitte mal https://github.com/gematik/api-erp/blob/e4233636137b2069d87a688c691fcb7f39418428/resources/scripts/build_with_replace_includes.sh
laufen lassen und das gerenderte Artefakt mit einchecken